src/PackageJsonSynchronizer.php

```diff
@@ -147,7 +147,7 @@ private function resolveImportMapPackages($phpPackage): array
         }
 
         $dependencies = [];
-
+        $entrypoints = $packageJson->read()['symfony']['entrypoints'] ?? [];
```

Comment:
Took some time to refresh my memory and re-read all this code 😅 

So .. nope : this is not related to importmap entrypoints, but to stimulus bundle + controllers.json file. Thus should not be used here :) 

Should not be used here